src/Symfony/Component/Validator/Constraints/TwigValidator.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
```

Comment:
I think, the constraint and its validator should be moved to the Twig Bridge and wired by TwigBundle.

Reply:
Thanks @derrabus  Moved to Twig Bridge and wired by TwigBundle